ui/packages/lib/src/index.scss

```diff
@@ -2,19 +2,13 @@
 @import "@elastic/eui/src/themes/amsterdam/globals";
 
 body {
-  @include euiHeaderAffordForFixed;
-
   padding-top: 0 !important;
 }
 
 #root {
   display: flex;
   flex-direction: column;
   min-height: 100vh;
-
-  .euiBody--headerIsFixed & {
-    min-height: calc(100vh - #{$euiHeaderHeightCompensation});
```

Comment:
was this actually unused all the while

Reply:
It was, but new update of EUI deprecate it.